src/main/java/fi/metatavu/famifarm/rest/V1RESTService.java

```diff
@@ -677,6 +725,19 @@ public Response findCampaign(UUID campaignId) {
     return createOk(campaignTranslator.translate(campaign));
   }
 
+  @Override
+  @RolesAllowed({ Roles.ADMIN, Roles.MANAGER, Roles.WORKER })
+  public Response findCutPacking(UUID cutPackingId) {
+    fi.metatavu.famifarm.persistence.model.CutPacking foundCutPacking = cutPackingController.find(cutPackingId);
+
+    if (foundCutPacking == null) {
+      return createNotFound("Cut packing with id " + cutPackingId + " not found!");
```

Comment:
Use formatter instead of concat i.e. String.format("Product with id %s not found!", productId)